src/Compiler/CMakeLists.txt

```diff
@@ -54,13 +54,32 @@ add_onnx_mlir_library(CompilerUtils
   MC
   )
 
+if( ACCELERATORS_TO_BUILD )
+  add_dependencies(CompilerUtils NNPAAccel)
```

Comment:
This would be better resolved as a list of libraries that just gets added above. I saw you have a function for adding the accelerator libraries - once that's cleaned up, it can automatically hold either NoAccel or the list of accelerator libraries to be added as one of the libs under `DEPENDS` for CompilerUtils.